### PR TITLE
2.0 thomas payan

### PR DESCRIPTION
**Pull request by Thomas23456**
Scaling:

- Conversion des features catégorielles au format numérique
- Scaling des features du dataset avec les méthodes suivantes : StandardScaler, MinMaxScaler, MaxAbsScaler, RobustScaler, PowerTransformer, QuantileTransformer, Normalizer

### CODE REVIEW

File: scripts/Scaling.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    def power_transformation(self, v_method='yeo-johnson'): # box-cox
+        print("\nPower transformation ("+v_method+")")
+        scaled_df = PowerTransformer(method=v_method).fit_transform(self.df)
+        scaled_df = self.convert_numpy_to_pandas(scaled_df)
+        return scaled_df
+    
+    # Function to Quantile transformation
+    def quantile_transformation(self, v_output_distribution='uniform'): # normal
+        print("\nQuantile transformation ("+v_output_distribution+")")
+        scaled_df = QuantileTransformer(output_distribution=v_output_distribution).fit_transform(self.df)
```
Comment (HerySon) [inline scripts/Scaling.py:80]: il faudrait rajouter à minima l'argument `n_quantiles` de `QuantileTransformer`

**Review by HerySon — CHANGES_REQUESTED** [review]

**Comment by HerySon** [comment]
@Thomas23456 

## Peux-tu:
- [ ] faire les corrections mineures suggérées ?

**Comment by Thomas23456** [comment]
Terminer l'intégration des **kwargs pour que l’utilisateur puisse implémenter les et surcharger les arguments des fonctions de transformation
